Includes/CubbyTower/Components/Type.hpp

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2021 CubbyTower Team
+// Chris Ohk, Minkyu Lee, Minjune Yi
+// We are making my contributions/submissions to this project solely in our
+// personal capacity and are not conveying any rights to any intellectual
+// property of any third parties.
+
+#ifndef CUBBYTOWER_TYPE_HPP
+#define CUBBYTOWER_TYPE_HPP
+
+namespace CubbyTower
+{
+//!
+//! \brief STATE struct.
+//!
+//! This struct stores whether the enemy's type is air/ground/stealth.
+//! 100 : Air, 010 : ground, 001 : stealth
+//!
+struct Type
+{
+    int type;
+};
```

Comment:
I think that this struct is not necessary.

Reply:
hmm.... why do you think so?

Isn't it necessary to save whether the monster is a ground/air/stealth unit?

Reply:
If I were you, I would create a structure called `Target` and check it with only variable `mask`.

Reply:
What exactly do Target and Mask specify?

Reply:
`Target` is a component that indicates an entity can be targeted.
`mask` is a variable that the type of entity's movement such as ground and air.

Reply:
I named 'Target' to 'TargetMask', 'mask' to 'typeMask'